persistence/sql/installer-workflow.md

```diff
@@ -67,9 +67,9 @@ Additionally, scripts assume the required databases and schemas already exist. E
 
 A sample workflow in that kind of environment can consist of the following steps:
 
- 1. In the project add references to both `NServiceBus.Persistence.Sql` and `NServiceBus.Persistence.Sql.MsBuild` packages.
+ 1. In the project, add references to the [necessary NuGet package(s)](/persistence/sql/#nuget-packages).
  1. Build Project.
- 1. `NServiceBus.Persistence.Sql.MsBuild` creates scripts in `bin\Debug\NServiceBus.Persistence.Sql`.
+ 1. SQL Persistence creates scripts in `bin\Debug\NServiceBus.Persistence.Sql`.
```

Comment:
I know @adamralph just went through and removed usages of "Persistence", so not sure if we should add one here or not.

Reply:
Also, the path mentioned here doesn't account for the `bin\Debug\{target}\NServiceBus.Persistence.Sql` variant.